docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -21,290 +21,1115 @@
+Integers and unsigned integers are parsed as Unix nanosecond timestamps and
+return the corresponding RFC3339 timestamp.
+
+```
+to_timestamp_micros(expression)
+```
+
+#### Arguments
+
+- **expression**: Column or literal value to operate on.
+
+### `to_timestamp_seconds`
+
+Converts a value to RFC3339 second timestamp format (`YYYY-MM-DDT00:00:00Z`).
+Supports timestamp, integer, and unsigned integer types as input.
+Integers and unsigned integers are parsed as Unix nanosecond timestamps and
+return the corresponding RFC3339 timestamp.
+
+```
+to_timestamp_seconds(expression)
+```
+
+#### Arguments
+
+- **expression**: Column or literal value to operate on.
 
 ### `from_unixtime`
 
-### `now`
+Converts an integer to RFC3339 timestamp format (`YYYY-MM-DDT00:00:00.000000000Z`).
+Input is parsed as a Unix nanosecond timestamp and returns the corresponding
+RFC3339 timestamp.
 
-Returns current time as `Timestamp(Nanoseconds, UTC)`. Returns same value for the function
-wherever it appears in the statement, using a value chosen at planning time.
+```
+from_unixtime(expression)
+```
 
-## Other Functions
+#### Arguments
 
-### `array`
+- **expression**: Column or integer literal to operate on.
 
-### `arrow_typeof`
+## Hashing Functions
 
-Returns the underlying Arrow type of the the expression:
+- [md5](#md5)
+- [sha224](#sha224)
+- [sha256](#sha256)
+- [sha384](#sha384)
+- [sha512](#sha512)
+
+### `md5`
 
-```sql
-❯ select arrow_typeof(4 + 4.3);
-+--------------------------------------+
-| arrowtypeof(Int64(4) + Float64(4.3)) |
-+--------------------------------------+
-| Float64                              |
-+--------------------------------------+
+Computes an MD5 128-bit checksum for a string expression.
+
+```
+md5(expression)
 ```
 
-### `in_list`
+#### Arguments
 
-### `random`
+- **expression**: Column or string literal to operate on.
 
 ### `sha224`
 
+Computes the SHA-224 hash of a binary string.
+
+```
+sha224(expression)
+```
+
+#### Arguments
+
+- **expression**: Column or string literal to operate on.
+
 ### `sha256`
 
+Computes the SHA-256 hash of a binary string.
+
+```
+sha256(expression)
+```
+
+#### Arguments
+
+- **expression**: Column or string literal to operate on.
+
 ### `sha384`
 
+Computes the SHA-384 hash of a binary string.
+
+```
+sha384(expression)
+```
+
+#### Arguments
+
+- **expression**: Column or string literal to operate on.
+
 ### `sha512`
 
+Computes the SHA-512 hash of a binary string.
+
+```
+sha512(expression)
+```
+
+#### Arguments
+
+- **expression**: Column or string literal to operate on.
+
+## Other Functions
+
+- [array](#array)
+- [arrow_typeof](#arrow_typeof)
+- [struct](#struct)
+
+### `array`
+
+Returns an Arrow array using the specified input expressions.
+
+```
+array(expression1[, ..., expression_n])
+```
+
+#### Arguments
+
+- **expression_n**: Column or literal value to include in the output array.
+
+### `arrow_typeof`
+
+Returns the underlying Arrow type of the the expression:
```

Comment:
I also noticed that `arrow_cast` is not documented (that was not introduced by this PR)